### PR TITLE
Remover logic

### PR DESCRIPTION
**Pull request by rebeccaL8**
Implementing logic to remove spec unit

### CODE REVIEW

File: Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs [hunk 1]
```diff
@@ -16,34 +16,37 @@ private static string RemoveCommandAndApplySemiColon(string clause, string searc
...
+                    clause = UpdateSimpleMehthod(clause, parenthesisIndex);
                 }
 
                 clauseStartIndex = clause.IndexOf(searchPattern);
             }
             return clause;
         }
 
-        private static string UpdateParameterMethod(string clause)
+        private static string UpdateParameterMethod(string clause, int parenthesisIndex, int indexOfComma)
```
Comment (RaphaelYoshiga) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs]: Which parenthesisIndex? the beginning or the end one?

File: Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover/TestCases/ExpectedMatfloAdapterShould.txt [hunk 2]
```diff
@@ -35,9 +35,9 @@ namespace X.Inventory.Stock.Iwt.IntegrationService.Tests.Unit.StockBuild
         public void Collaborate_with_matflo_api_to_send_heartbeat_only_when_no_previous_heartbeat_sent()
         {
             i_have_an_instance_of_matflo();
-            i_send_a_heartbeat_request();
-            i_send_a_heartbeat_request);
-            we_only_call_matfloApi_0_(1, "heartbeat");
+				i_send_a_heartbeat_request();
```
Comment (RaphaelYoshiga) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover/TestCases/ExpectedMatfloAdapterShould.txt]: This looks like cheating, it was forcing to put the methods on the place visual studio will put as soon as we press CRLT+ K + D

File: Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs [hunk 3]
```diff
@@ -16,34 +16,37 @@ private static string RemoveCommandAndApplySemiColon(string clause, string searc
...
+
+            var endIndex = result.IndexOf(")", indexOfComma);
+            if (result.IndexOf(";",endIndex) == endIndex +1)
+                return result;
+
+            return result.Insert(parenthesisIndex + 1, SEMICOLON);
         }
 
-        private static string UpdateSimpleMehthod(string value, int index)
+        private static string UpdateSimpleMehthod(string value, int parenthesisIndex)
```
Comment (RaphaelYoshiga) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs]: Small mistype

File: Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs [hunk 4]
```diff
@@ -16,34 +16,37 @@ private static string RemoveCommandAndApplySemiColon(string clause, string searc
             var clauseStartIndex = clause.IndexOf(searchPattern);
             while (clauseStartIndex > 0)
             {
-                var parametersIndex = clause.IndexOf(",");
```
Comment (RaphaelYoshiga) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs]: This method is a little bit greedy, extracting looks necessary

File: Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover/TestCases/ExpectedMatfloAdapterShould.txt (hunk 2 above)
Comment (rebeccaL8) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover/TestCases/ExpectedMatfloAdapterShould.txt]: I thought to do a simpler scenario before trying to fix the indenting

Comment (RaphaelYoshiga) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover/TestCases/ExpectedMatfloAdapterShould.txt]: Hmm, I understood, as this is going to be the acceptance test (Final test), I would let with the expected identation

File: Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs [hunk 5]
```diff
@@ -2,6 +2,9 @@
 {
     public class ContentRemover
     {
+        private static int _endParenthesisIndex;
```
Comment (RaphaelYoshiga) [inline Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs]: on this project this won't be a problem, but do you know the true implications of marking this as static?

**Pull request merged**